src/Gateway/PayplugGateway.php

```diff
@@ -637,22 +637,13 @@ public function scripts()
 		// Register checkout styles.
 		wp_register_style('payplug-checkout', PAYPLUG_GATEWAY_PLUGIN_URL . 'assets/css/payplug-checkout.css', [], PAYPLUG_GATEWAY_VERSION);
 		wp_enqueue_style('payplug-checkout');
-		//$ip_was_activated = false;
 
-		$ip_was_activated = $this->activate_integrated_payments();
-		$transient_key = PayplugWoocommerceHelper::get_transient_key(get_option('woocommerce_payplug_settings', []));
-		$ip = get_transient($transient_key);
-		if( $ip_was_activated || ((isset($ip['permissions']['can_use_integrated_payments']) && ($ip['permissions']['can_use_integrated_payments'] === true)) && ($this->payment_method == "integrated")) ){
+		if ($this->payment_method == "integrated" ) {
 			$this->integrated_payments_scripts();
-			$ip_was_activated = true;
-
 		}
 
-		if ($this->payment_method == "popup" && !$ip_was_activated && $this->id === "payplug") {
+		if ($this->payment_method == "popup" ) {
```

Comment:
you should keep the `$this->id === "payplug"` 
this will run x times has many gateways you've, so you'll register the scripts x times to... but if you import this only for the `payplug´ gateways, this will only run once ;)